bluepyemodel/model/model.py

```diff
@@ -256,10 +258,14 @@ def define_morphology(
                 try:
                     modifier_module = importlib.import_module(morph_modifier[0])
                     morph_modifiers[i] = getattr(modifier_module, morph_modifier[1])
-                    if len(morph_modifier) > 2:
+                    if len(morph_modifier) == 3:
                         morph_modifiers_hoc[i] = getattr(modifier_module, morph_modifier[2])
+                    elif len(morph_modifier) == 2:
+                        morph_modifiers_hoc[i] = getattr(
+                            modifier_module, morph_modifier[1] + "_hoc"
+                        )
                     else:
-                        morph_modifiers_hoc[i] = getattr(modifiers, morph_modifier[1] + "_hoc")
+                        raise AttributeError()
```

Comment:
Could you add an error message here?

Reply:
I have refactored the error logging, let me know if it is more clear